README.md

```diff
@@ -107,45 +107,68 @@ Requirements:
 
 * Ruby > 3.2.2
 * [Bundler](http://bundler.io/)
-* [Gem Release](https://github.com/svenfuchs/gem-release)
 * [Rake](https://github.com/ruby/rake)
-* A [RubyGems](https://rubygems.org) account that has
-  [ownership](http://guides.rubygems.org/publishing/) of the gem
 * RuboCop and the unit tests should all pass (`rake`)
 
-The [CHANGELOG.md](CHANGELOG.md) is not automatically updated by the
-`rake release:*` tasks. Update it manually to insert the correct version and
-date, and commit the file, before creating the release package.
+To make a release, follow theses steps:
 
-**gem release** is used by rake tasks in this project to handle version changes,
-tagging, and publishing to RubyGems.
+1. Confirm no errors are returned by `bundle exec rubocop` *
+2. Confirm all tests pass (`bundle exec rake spec:unit`) *
+3. Ensure that the `CHANGELOG.md` file is up to date with an `Unreleased`
+   header.
+4. Submit a Pull Request, and merge the branch into `main`
+5. On your local machine, ensure you are on the `main` branch (and have
+   it up-to-date), and run `bundle exec rake bump:*` (see below)
+6. This will trigger the GitHub Actions CI workflow to push a release to
```

Comment:
I don't think this needs to be a separate item -- the statement pertains to step 5.